README.md

```diff
@@ -5,52 +5,54 @@ Feel free to check out the full project on [GitHub](https://github.com/Tech-TTGa
 * [Database Information](docs/database_info.html)
 
 ## Info
-A Discord bot that adds extensions to the [Tickets](https://github.com/TicketsBot) discord bot*<sup>[1]</sup>.
-Which works as a supplimentary, separate bot*<sup>[2]</sup>.
+A Discord bot that adds extensions to the [Tickets](https://github.com/TicketsBot) discord bot<sup>[1]</sup>.
+Which works as a supplimentary, separate bot<sup>[2]</sup>.
 
-<sub>[1]*When a feature is added to the main bot it will be removed from here.<sub>
+<sub>[1] - When a feature is added to the main bot it will be removed from here.<sub>
 
-<sub>[2]*Note: This setup assumes you have the main bot added to the server and configured, Support will not be provided to those who do not assume a likewise configuraton.</sub>
+<sub>[2] - Note: This setup assumes you have the main bot added to the server and configured, Support will not be provided to those who do not assume a likewise configuraton.</sub>
 
 ## Feature 1 - Staff Notes
 
 Private threads are *(now)* free!
 
-This creates a private thread in a channel and adds the staff roles to it*.
```

Comment:
I agree with the changes made in the first footnote sector. 
However I believe that * should be used opposed to <sup>[#]</sup> when there is only one footnote added.

Reply:
To add, this would also apply to the other sector where only one footnote is added, however I forgot to add a comment to said sector